src/mo_frontend/parser.mly

```diff
@@ -162,7 +162,8 @@ let share_exp e =
 
 let share_dec d =
   match d.it with
-  | LetD (p, e) -> LetD (p, share_exp e) @? d.at
+  | LetD (p, e, None) -> LetD (p, share_exp e, None) @? d.at
```

Comment:
Fix this:
`actor { public let x = shared func f() {} else { loop() }; };`

Reply:
But here the `else ...` part is dead, right?